app.js

```diff
@@ -22,14 +22,14 @@ app.use(
   )
 );
 
-morgan.token("user", function(req) {
+morgan.token("user", function (req) {
```

Comment:
Travis checks have failed due to changes in this file. Revert this file to previous state.